config/manifest-loader.js

```diff
@@ -7,8 +7,8 @@ function manifestLoader(source) {
   return `
     ${source}
     (function() {
-      if (module.exports.receiveContentProps) {
-        const applicationSettings = window.settings.applications[module.exports.entryName];
+      const applicationSettings = window.settings.applications[module.exports.entryName];
```

Comment:
@ncksllvn Does this change make sense to you? The dashboard code imports manifests for some forms that aren't in production, which means they don't have settings values. This skips running `receiveContentProps` when that is the case. Should be pretty rare outside of applications with cross-cutting features like the dashboard.

Reply:
Ah that is a tricky case. That does make sense to me! Appreciate the heads up on that.